src/extension/wps/src/main/java/au/org/emii/aggregator/NetcdfAggregator.java

```diff
@@ -120,8 +123,9 @@ private void assertOutputPathValid(Path outputPath) {
 
     private void copyToOutputFile(NetcdfDatasetIF template) throws AggregationException {
         try {
-            writer = NetcdfFileWriter.createNew(NetcdfFileWriter.Version.netcdf4,
-                outputPath.toString(), null);
+            Nc4Chunking chunking = new Nc4ChunkingDefault(3, false);
```

Comment:
If the code is to be used by someone else, then it will be good if we make it configurable and move it to gogoduck.xml 


Does that make sense?